lib/child_process.js

```diff
@@ -949,6 +955,15 @@ function execSync(command, options) {
 }
 
 
+function validateArgumentsNullCheck(args) {
+  for (let i = 0; i < args.length; ++i) {
+    if (StringPrototypeIncludes(args[i], '\u0000')) {
+      throw new ERR_INVALID_ARG_VALUE('args', args, 'cannot contain null bytes');
```

Comment:
Consider changing the error message to `must be a string without null bytes` for consistency with errors from the fs module.

Reply:
@bnoordhuis done, PTAL!